harmonizer/src/js_types.rs

```diff
@@ -63,10 +63,11 @@ impl Error for CompositionError {}
 /// the `extensions` property of a JavaScript [`GraphQLError`] from the
 /// [`graphql-js`] library. Such errors are created when errors have prevented
 /// successful composition, which is accomplished using [`errorWithCode`]. An
-/// [example] of this can be seen within the `federation-js` JavaScript library.
+/// [example] of this can be seen within the `composition-js` JavaScript library.
 ///
 /// [`graphql-js']: https://npm.im/graphql
 /// [`GraphQLError`]: https://github.com/graphql/graphql-js/blob/3869211/src/error/GraphQLError.js#L18-L75
+/// TODO: replace with `composition-js` equivalents?
```

Comment:
Wasn't sure here

Reply:
Pushed an additional commit that modifies those (that, for completeness sake, ensuring we ship codes will all errors and documenting them is one of the TODO for federation 2 before GA, but "some" of the errors do get code already and I've link to an example of those).